utils/src/lib.rs

```diff
@@ -688,6 +689,82 @@ pub fn abi_decode(sig: &str, calldata: &str, input: bool) -> Result<Vec<Token>>
     Ok(res)
 }
 
+#[derive(Default, Serialize)]
+pub struct RawSelectorImportData {
+    pub function: Vec<String>,
+    pub event: Vec<String>,
+    pub error: Vec<String>,
+}
+
+#[derive(Serialize)]
+#[serde(untagged)]
+pub enum SelectorImportData {
+    Abi(Vec<LosslessAbi>),
+    Raw(RawSelectorImportData),
+}
+
+#[derive(Serialize)]
+struct SelectorImportRequest {
+    #[serde(rename = "type")]
+    import_type: String,
+    data: SelectorImportData,
+}
```

Comment:
can we move all of these to their own module? like selectors.rs or smth

Reply:
yeah, good idea

Reply:
done in https://github.com/foundry-rs/foundry/pull/1716/commits/3d14fdd10e65e213d0b3f67d6abe276abb34ef12